README.md

```diff
@@ -57,6 +57,7 @@ Check out the [configuration part](https://thundra.readme.io/docs/nodejs-configu
 | thundra_lambda_warmup_warmupAware        |  bool  |     false     |
 | thundra_lambda_trace_request_skip        |  bool  |     false     |
 | thundra_lambda_trace_response_skip       |  bool  |     false     |
+| thundra_lambda_timeout_margin            | number |     200       |
```

Comment:
HTTP response check skip related environment variable should also be added here with default value is `false`

Reply:
👍

Reply:
ok I see :+1: